### DISCUSSION TITLE
DateRangePicker Locale

### DISCUSSION THREAD

**Discussion by ramiroguzmanc** [tremorlabs/tremor-npm#780] [Q&A] [answered]
Hello, 

I want to use the DateRangePicker component in spanish. In the documentations says that I can use locale importing the locale from `date-fns`. But when I import { es } from date-fns/locale and use it on locale param, only change the languaje from months and days. But the rest of the component stays in english. 

<img width="297" alt="image" src="https://github.com/tremorlabs/tremor/assets/38337109/3416bd25-c474-485d-9105-9f38ae00975c">
<img width="300" alt="image" src="https://github.com/tremorlabs/tremor/assets/38337109/1000f3de-2980-4119-83fc-54d129c1a25e">

**Comment by mbauchet** [comment, marked as answer]
You have two placeholder props to set your own text. And you can add your own options on select
